modules/terraform-aws-vpc/variables.tf

```diff
@@ -12,6 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+variable "create_vpc" {
+  type = bool
+
+  description = "Flag to specify if the VPC should be created. Otherwise data sources will be used to lookup details of existing resources."
+
+  default = true
+}
+
 variable "vpc_name" {
```

Comment:
@jimright Some description could be added here as well, also pointing out that this input is only used when create\_vpc is false.

Reply:
Done.